KeeTrayTOTP/Menu/EntryMenuItemProvider.cs

```diff
@@ -23,9 +23,13 @@ public ToolStripMenuItem ProvideMenuItem()
         {
             var rootEntryMenuItem = new ToolStripMenuItem(Localization.Strings.TrayTOTPPlugin, Properties.Resources.TOTP);
 
-            var entryMenuCopyTotp = new ToolStripMenuItem(Localization.Strings.CopyTOTP, Properties.Resources.TOTP, OnEntryMenuTOTPClick, (Keys)Shortcut.CtrlT);
-            var entryMenuSetupTotp = new ToolStripMenuItem(Localization.Strings.SetupTOTP, Properties.Resources.TOTP_Setup, OnEntryMenuSetupClick, (Keys)Shortcut.CtrlShiftI);
-            var entryMenuShowQrCode = new ToolStripMenuItem(Localization.Strings.ShowQR, Properties.Resources.TOTP_Setup, OnEntryMenuShowQRClick, (Keys)Shortcut.CtrlShiftJ);
+            var entryMenuCopyTotp = new ToolStripMenuItem(Localization.Strings.CopyTOTP, Properties.Resources.TOTP, OnEntryMenuTOTPClick);
+            var entryMenuSetupTotp = new ToolStripMenuItem(Localization.Strings.SetupTOTP, Properties.Resources.TOTP_Setup, OnEntryMenuSetupClick);
+            var entryMenuShowQrCode = new ToolStripMenuItem(Localization.Strings.ShowQR, Properties.Resources.TOTP_Setup, OnEntryMenuShowQRClick);
+
+            entryMenuCopyTotp.ShortcutKeys = (Keys)Shortcut.CtrlT;
```

Comment:
Maybe explicitly mention the mono issue here, so that we don't fall into the same trap.

This should still work and looks a bit cleaner to me. What do you think?

```csharp
var entryMenuCopyTotp = new ToolStripMenuItem(Localization.Strings.CopyTOTP, Properties.Resources.TOTP, OnEntryMenuTOTPClick) {
   ShortcutKeys = (Keys)Shortcut.CtrlT;
}
```

Reply:
Yeah this looks better. At first I wanted to use the if directive for mono, but this could be a problem if we release a dll at some point.
```csharp
#if __MonoCS__
    entryMenuCopyTotp.ShortcutKeys = (Keys)Shortcut.CtrlT;
    ...
#endif
```

Reply:
Not really a fan of putting too much knowledge of the environment so if we can prevent it like this I'd favour that.